package.json

```diff
@@ -31,11 +31,12 @@
         "/src/ipcEventsAddon.py"
     ],
     "scripts": {
-        "build": "parcel build",
+        "build": "yarn print-version && parcel build",
         "fix": "yarn eslint . --ignore-path .gitignore --ext .js,.jsx,.ts,.tsx --fix",
         "postinstall": "node scripts/postinstall.js",
         "lint": "tsc && eslint . --ignore-path .gitignore --ext .js,.jsx,.ts,.tsx && git diff --check",
-        "prepack": "rm -rf dist && yarn build && yarn typedoc",
+        "prepack": "rm -rf dist && yarn build && yarn typedoc && git add src/version.gen.ts",
```

Comment:
Wait, why would we `git add` in `prepack`?

Reply:
Because if we increment the version we want the file to be added to the version commit. But actually, this belongs into postversion now that I think more about it.

Reply:
`postversion` definitely seems more sensible to me. But that should also do `yarn print-version` then.

Reply:
Yes, I included this. I still think we should do this in build, too, though.